ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/FeeEstimate.hs

```diff
@@ -0,0 +1,143 @@
+-- | Calculate fee estimates for new transaction
+module  Ariadne.Wallet.Cardano.Kernel.FeeEstimate
+        ( transactionInputs
+        , cardanoFee
+        ) where
+
+import GHC.Show (show)
```

Comment:
Why not `Text.Show`?

Reply:
I suppose simply because it's imported code. Probably `Text.Show` is better, because I don't even see `GHC.Show` [here](https://www.stackage.org/lts-11.2/package/base-4.10.1.0). Will change.